src/migrate/Migrator.js

```diff
@@ -376,7 +345,7 @@ export default class Migrator {
           );
         })
         .then(() => {
-          log.push(path.join(directory, name));
+          log.push(name);
```

Comment:
This change is in the other PR, mainly because I don't have this info anymore. And I can't add the migration directory into the migration name.

I could add a `getMigrationId()` vs `getMigrationName()`. Id would return what is stored in the knex migration table, and the name could be logged / used for diagnostics.

Thoughts?